app/themes/base.js

```diff
@@ -61,21 +48,18 @@ export default {
     grey600,
     grey700,
     grey800,
-    grey900
-  ],
-  colorText: darkBlack,
-  colorAlternateText: white,
-  colorCanvas: white,
-  colorBorder: grey300,
-  colorShadow: fullBlack,
-  spacingNone: 0,
-  spacingExtraSmall: '.25rem',
-  spacingSmall: '.5rem',
-  spacingMedium: '1rem',
-  spacingLarge: '2rem',
-  spacingExtraLarge: '4rem',
-  spacingExtraExtraLarge: '8rem',
-  spacingExtraExtraExtraLarge: '16rem',
-  borderRadius: '2px',
```

Comment:
do we want to get rid of these parts of the theme?

- `borderRadius`
- `zIndexMax`
- `em`
- `rem`